src/arrayMethodJoin.js

```diff
@@ -4,9 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(sep = ',') {
```

Comment:
Don't use shortened words as variable names.
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```